DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/AccessoAllaPiattaforma.tex

```diff
@@ -1,58 +1,290 @@
+	\item \textbf{Precondizione:} l'utente non è ancora presente nella piattaforma;
+	\item \textbf{Postcondizione:} l'utente è registrato con un account acquirente ed è autenticato come tale sulla piattaforma;
+	\item \textbf{Scenario principale:}
+	\begin{itemize}
+		\item L'utente accede al sistema;
+		\item L'utente non autenticato si trova nella schermata di registrazione;
+		\item (UC) - Inserisce tutti i dati per compilare il modulo di registrazione;
+		\item L'utente non autenticato conferma il proprio indirizzo email inserito;
+		\item L'utente possiede un account presso il sistema.
+	\end{itemize}
+	\item \textbf{Estensioni:}
+	\begin{enumerate}
+		\item L'utente inserisce un indirizzo email e l'identity manager segnala che è già presente nella piattaforma
+		\begin{itemize}
+			\item L'utente non viene registrato presso il sistema;
+			\item (UC) - Visualizzazione messaggio di errore in caso di email già registrata nella piattaforma;
+			\item Viene fornita all'utente la possibilità di modificare il proprio indirizzo email inserito.
+		\end{itemize}
+		\item L'utente compila i campi dati per la password e la conferma della stessa con due password diverse, in questo caso:
+		\begin{itemize}
+			\item L'utente non viene registrato presso il sistema;
+			\item L'identity manager segnala che le due password inserite non coincidono;
+			\item (UC) - Visualizzazione messaggio di errore in caso di password e password di conferma diverse;
+			\item L'utente può modificare le password inserite.
+		\end{itemize}
+		%\item l'utente prova ad accedere alla piattaforma con i dati inseriti nel modulo senza aver confermato il proprio indirizzo email
+		%\begin{itemize}
+		%	\item (UC) - Visualizzazione messaggio di errore in caso di credenziali sconosciute;
+		%	\item L'utente non possiede un account presso il sistema.
+		%\end{itemize}
+	\end{enumerate}
+\end{itemize}
+
+\subUC{Inserimento dati per la registrazione}
+L'utente compila il modulo per proseguire con la registrazione presso la piattaforma.
+\begin{itemize}
+	\item \textbf{Attori primari:} utente non autenticato;
+	\item \textbf{Attori secondari:} identity manager;
+	\item \textbf{Precondizione:} l'utente non è autenticato e si trova all'interno della schermata di registrazione;
+	\item \textbf{Postcondizione:} l'utente ha compilato il modulo e può procedere con la registrazione;
+	\item \textbf{Scenario principale:} l'utente non autenticato compila i campi nel seguente modo:
+	\begin{itemize}
+		\item (UC) - Inserimento nome per la registrazione;
+		\item (UC) - Inserimento cognome per la registrazione;
+		\item (UC) - Inserimento indirizzo e-mail per la registrazione;
+		\item (UC) - Inserimento password per la registrazione;
+		\item (UC) - Inserimento conferma password.
+	\end{itemize}
+\end{itemize}
+
+\subUC{Inserimento del nome per la registrazione}
+\begin{itemize}
+	\item \textbf{Attori primari:} utente non autenticato;
+	\item \textbf{Attori secondari:} identity manager;
```

Comment:
Per me togliere attore secondario negli inserimenti, visto che non fa nulla in questo caso